datepicker/_datepicker.scss

```diff
@@ -21,6 +21,7 @@ $size: 2.5em;
   }
 
   &__popup {
+    background-color: white;
```

Comment:
This fails our Sass linting; no color literals (so `#fff` instead of `white`) and no direct colors. Simply changing this to `$white` to re-use the white variable already in place will resolve this issue, and this is good to merge!